WorkflowTesting/Sources/Internal/AppliedAction.swift

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Workflow
+import XCTest
+
+struct AppliedAction<WorkflowType: Workflow> {
```

Comment:
Does this need to be wrapped in an `#if DEBUG`?

Reply:
I don’t _think_ so if the comment about `@testable import` being the issue was correct (since this file does a standard import)